code/__DEFINES/pipes.dm

```diff
@@ -1,4 +1,6 @@
-//Atmospherics pipes
+//////////////////////////////
+// MARK: Atmospherics pipes
+//////////////////////////////
```

Comment:
Please check out the devdocs [Here](https://devdocs.paradisestation.org/coding/style_guidelines/#mark-comments) to see about the style of these comments!